xcode/Subconscious/Shared/Components/AppView.swift

```diff
@@ -2431,22 +2498,169 @@ struct AppModel: ModelProtocol {
         )
     }
     
-    static func notifySucceedUnfollow(
+    static func followPeer(
```

Comment:
All this logic has been lifted from `UserDetailModel` to the app level. This simplifies notification routing and means we can trigger these actions from any view.